packages/qdrant-loader/src/qdrant_loader/core/pipeline/orchestrator.py

```
@@ -190,17 +198,36 @@ async def _process_all_projects(
+                    aggregated_result.successfully_processed_documents.update(
+                        project_result.successfully_processed_documents
+                    )
+                    aggregated_result.failed_document_ids.update(
+                        project_result.failed_document_ids
+                    )
+                    aggregated_result.errors.extend(project_result.errors)
+
                 logger.debug(
                     f"Processed {len(project_documents)} documents from project: {project_id}"
                 )
             except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)